cypress/integration/ui/dashboard_filter/dashboard_filter.js

```diff
@@ -0,0 +1,26 @@
+import { When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+const OPTIONS = { timeout: 15000 }
```

Comment:
I think this can be moved to a global util prop to be reused, e.g. `.utils/options.js` with `export const extendedTimeout = ....`